packages/web-runtime/src/services/uppyService.ts

```diff
@@ -51,7 +51,8 @@ export class UppyService {
       chunkSize: chunkSize,
       removeFingerprintOnSuccess: true,
       overridePatchMethod: !!tusHttpMethodOverride,
-      retryDelays: [0, 500, 1000]
+      retryDelays: [0, 500, 1000],
+      uploadDataDuringCreation: true
```

Comment:
can we use capabilities here? the tus plugin is announced in the capabilities AFAIR.